website/docs/repl.md

```diff
@@ -17,6 +17,22 @@ This would open a Chrome browser that you can control with the REPL interface. M
 wdio repl chrome -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY
 ```
 
+If driver is running on different port eg : 9515, it could passed with the command line argument --port or alias -p
+
+```sh
+wdio repl chrome -u $SAUCE_USERNAME -k $SAUCE_ACCESS_KEY -p 9515
+```
+
+Repl could also be ran using the capabilities from the webdriverIO config file. Wdio supports capabilities object; or ; multiremote capability list or object. 
+
+If the config file uses capabilities object then just pass the path to config file, else if its a multiremote capability then, specify which capability to use from list or multiremote using the positional argument . Note: for list we consider zero based index. 
+
+```sh
+wdio repl "./path/to/wdio.config.js" 
+wdio repl "./path/to/wdio.config.js" 0 -p 9515
+wdio repl "./path/to/wdio.config.js" "capabilityPropertyName" -p 9515
```

Comment:
Maybe we add some examples, e.g. "if your config capabilities look like this you can run a repl for this capability by executing this".